### PR TITLE
feat: add settings view for url 

### PR DESCRIPTION
**Pull request by lislis**
Adding a route and view for saving the url to localStorage.

Screenshot_2020-03-28 mete0rama-orama

### CODE REVIEW

File: mete0rama-orama/src/main.js [hunk 1]
```diff
@@ -3,20 +3,16 @@ import App from './App.vue'
...
 new Vue({
   router,
   store,
   mounted () {
-    axios
-      .get('http://192.168.178.42/data')
-      .then(response => {
-        store.commit('updateMeasurements', response.data.data)
-      })
+    store.dispatch('fetchStationURL').then(() => {
```
Comment (vsmart) [inline mete0rama-orama/src/main.js]: Nitpick: `then` usually goes on the next line.

File: mete0rama-orama/src/store/index.js [hunk 2]
```diff
@@ -1,18 +1,47 @@
...
     }
   },
   actions: {
+    async fetchMeasurements ({ commit, state }) {
+      const endpoint = state.stationEndpoint
+      if (endpoint !== '') {
+        axios
+          .get(endpoint)
+          .then(response => {
+            commit('SET_MEASUREMENTS', response.data.data)
```
Comment (vsmart) [inline mete0rama-orama/src/store/index.js]: Is the commit also an async call? If yes, we should add another `then` before resolving.

Comment (lislis) [reply mete0rama-orama/src/store/index.js:27]: no, committing the mutation is a synchronous call

File: mete0rama-orama/src/components/StationSetting.vue [hunk 3]
```diff
@@ -2,15 +2,10 @@
   <section>
     <h2 class="settings-header">Station URL</h2>
     <p class="form-help">This is the data endpoint that your Mete0rama station exposes in your home network.</p>
-    <div v-if="hasStationURL">
-      <p>You're currently fetching from <br>
-        <span class="highlight">{{stationURL}}</span></p>
+    <div>
+      <p v-if="hasStationURL">You're currently fetching from</p>
+      <p v-else>Please enter a station URL</p>
```
Comment (vsmart) [inline mete0rama-orama/src/components/StationSetting.vue:7]: add a `:`

File: mete0rama-orama/src/components/StationSetting.vue [hunk 4]
```diff
@@ -2,15 +2,10 @@
   <section>
     <h2 class="settings-header">Station URL</h2>
     <p class="form-help">This is the data endpoint that your Mete0rama station exposes in your home network.</p>
-    <div v-if="hasStationURL">
-      <p>You're currently fetching from <br>
-        <span class="highlight">{{stationURL}}</span></p>
+    <div>
+      <p v-if="hasStationURL">You're currently fetching from</p>
```
Comment (vsmart) [inline mete0rama-orama/src/components/StationSetting.vue:6]: add a `:`

**Pull request merged**